src/coreclr/vm/domainfile.cpp

```diff
@@ -1894,137 +1880,6 @@ void DomainAssembly::SetupDebuggingConfig(void)
 #endif // DEBUGGING_SUPPORTED
 }
 
-// The format for the (temporary) .INI file is:
-
-// [.NET Framework Debugging Control]
-// GenerateTrackingInfo=<n> where n is 0 or 1
-// AllowOptimize=<n> where n is 0 or 1
-
-// Where neither x nor y equal INVALID_INI_INT:
-#define INVALID_INI_INT (0xFFFF)
-
-bool DomainAssembly::GetDebuggingOverrides(DWORD *pdwFlags)
-{
-    CONTRACTL
-    {
-        INSTANCE_CHECK;
-        THROWS;
-        GC_NOTRIGGER;
-        MODE_ANY;
-        INJECT_FAULT(COMPlusThrowOM(););
-    }
-    CONTRACTL_END;
-
-#if defined(DEBUGGING_SUPPORTED) && !defined(FEATURE_CORESYSTEM)
```

Comment:
`FEATURE_CORESYSTEM` is always defined, so this is never used. Found this while reviewing the hot reload change.